mappings/servers/hvbc/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "hvbc",
     "name": "HVBC",
-    "description": "Practice Server For Legit player",
+    "description": "Practice Server For Legitimate player",
```

Comment:
A few more grammar tweaks I recommend

Descriptions should not be title case (“Practice Server for Legitimate Player” -> “Practice server for legitimate player”)

pretty sure “player” seems to be intended as a plural use. I would just use “players”

in descriptions, typically you don’t want an abrupt start. That’s why I had “A” at the start